server/channels/app/imports/import_validators.go

```diff
@@ -312,6 +312,32 @@ func ValidateUserImportData(data *UserImportData) *model.AppError {
 	return nil
 }
 
+func ValidateBotImportData(data *BotImportData) *model.AppError {
+	if data.ProfileImage != nil && data.ProfileImageData == nil {
+		if _, err := os.Stat(*data.ProfileImage); os.IsNotExist(err) {
```

Comment:
So we are returning with an error if the file doesn't exist, but what about other error cases opening the file?